src/erlcloud_athena.erl

```diff
@@ -566,6 +632,16 @@ get_url(#aws_config{athena_scheme = Scheme,
                     athena_port   = Port}) ->
     Scheme ++ Host ++ ":" ++ integer_to_list(Port).
 
-update_description(Request, undefined)   -> Request;
-update_description(Request, Description) ->
-    maps:put(<<"Description">>, Description, Request).
+encode_params(Params) ->
+  encode_params(Params, []).
+
+encode_params([], Acc) ->
+  maps:from_list(Acc);
+encode_params([{_, undefined} | T], Acc) ->
+  encode_params(T, Acc);
+encode_params([{description, Description} | T], Acc) when is_binary(Description) ->
+  encode_params(T, [{<<"Description">>, Description} | Acc]);
+encode_params([{workgroup, WorkGroup} | T], Acc) when is_binary(WorkGroup) ->
```

Comment:
optional:
I would eliminate this atom to binary conversion and use binary directly. but optional as other modules here often do this

Reply:
I wanted to make this module more consistent with other modules. Sort of less thinking required from the usage perspective.